README.rst

```diff
@@ -61,7 +61,7 @@ Optional Settings
 .. code-block:: python
 
 	GRAPH_AUTH = {
-	    'USER_FIELDS': ('email', 'first_name', 'last_name', ), # Which username fields are available
+	    'USER_FIELDS': ('username', 'email', 'first_name', 'last_name', ), # Which user fields are available
```

Comment:
I don't think we should have this as a default. Instead, a note would be appropriate.